src/libNode/Node.h

```diff
@@ -294,7 +294,8 @@ class Node : public Executable {
   bool CheckMicroBlockVersion();
   bool CheckMicroBlockshardId();
   bool CheckMicroBlockTimestamp();
-  bool CheckMicroBlockHashes(bytes& errorMsg);
+  bool CheckMicroBlockHashes(bytes& errorMsg,
+                             const uint64_t& microblock_gas_limit);
```

Comment:
Seperate the check gas limit to another function?

Reply:
Fixed